src/crafting.cpp

```diff
@@ -1185,7 +1185,7 @@ bool player::disassemble( item &obj, int pos, bool ground, bool interactive )
 void player::disassemble_all( bool one_pass )
 {
     // Reset all the activity values
-    assign_activity( ACT_DISASSEMBLE, 0 );
+    assign_activity( activity_id( "ACT_DISASSEMBLE" ), 0 );
```

Comment:
`assign_activity` should probably have default `time` as either 0 or calendar indefinite